tests/model_registry/conftest.py

```diff
@@ -330,7 +330,6 @@ def get_call_kwargs(self, case: Case) -> dict[str, Any]:
 
         def after_call(self, response: Response, case: Case) -> None:
             LOGGER.info(f"{case.method} {case.path} -> {response.status_code}")
-            # LOGGER.info("This is the after_call function")
```

Comment:
This seems like leftover from debugging. Can you please remove?

Reply:
It is already removed @dbasunag